### PR TITLE
Improved: Search on mobile (#542)

### LINKED ISSUE #542: iOS zooms into input fields
## Current behavior

Font size is less than 16px, due to which safari zooms into that input box

## Expected behavior

We can inc the size to base

## Steps to reproduce the issue

1. Open capaybara website on ios mobile
2. Click on search 
3. Select input box of search

## Can you handle fixing this bug by yourself?

- [X] YES
- [ ] NO

## Environment details

- Browser:  Safari
- OS:  iOS
- Code Version:  All

### PR DESCRIPTION
**Pull request by azkyakhan**
### Related Issues
#542 

Closes #542 

### Short Description and Why It's Useful
Improved font side of search text and also no result margin

### Screenshots of Visual Changes before/after (If There Are Any)
Search-before
Search-after

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and Currently Important Rules Acceptance

- [X] I read and followed [contribution rules](https://github.com/DivanteLtd/vsf-capybara/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: components/organisms/o-search-panel.vue [hunk 1]
```diff
@@ -239,10 +239,15 @@ export default {
   }
 
   .no-results {
-    height: 5rem;
-    line-height: 5rem;
     display: flex;
     justify-content: center;
+    height: 5rem;
+    line-height: var(--spacer-base);
```
Comment (dt2patel) [inline components/organisms/o-search-panel.vue]: Line height should not be set using spacer. Spacers should only used for spacing related use cases like: gap, padding and margin.

Comment (dt2patel) [reply components/organisms/o-search-panel.vue]: For this issue I think just changing font size is enough. don't need anything else @azkyakhan

**Review by dt2patel — APPROVED** [review]
Thanks Azkya

File: components/organisms/o-search-panel.vue (hunk 1 above)
Comment (azkyakhan) [reply components/organisms/o-search-panel.vue]: Ok done sir

**Pull request merged**